app/BUILD.bazel

```diff
@@ -490,28 +492,28 @@ kt_android_library(
         artifact("androidx.databinding:databinding-common"),
         artifact("androidx.databinding:databinding-runtime"),
     ],
-    visibility = ["//visibility:private"],
 )
 
 # Library for scope annotations required to build views and view_model libraries.
 kt_android_library(
     name = "annotations",
-    custom_package = "org.oppia.app",
     srcs = ANNOTATIONS,
+    custom_package = "org.oppia.app",
+    visibility = ["//visibility:private"],
     deps = [
         ":dagger",
         "//model",
     ],
-    visibility = ["//visibility:private"],
 )
 
 # Library for all view model files.
 kt_android_library(
     name = "view_models",
-    custom_package = "org.oppia.app.view.models",
     srcs = VIEW_MODELS,
+    custom_package = "org.oppia.app.view.models",
     enable_data_binding = True,
     manifest = "src/main/ViewModelsManifest.xml",
+    visibility = ["//visibility:private"],
```

Comment:
Here and above: this should be default.

Reply:
I didn't get this. 
Do you mean, we should remove this as defaults to private, and also remove the package(default_visibility = ...)?

Reply:
I think we should keep package(default_visibility) and remove these. But the package default visibility should not be `//visibility:public`
Instead it should be a list of packages that have this as a dependency (or private in case its all private)

Reply:
Solved in the new PR due to a lot of conflicts in this PR - https://github.com/oppia/oppia-android/pull/2048